dpytools/embeds.py

```diff
@@ -45,3 +45,19 @@ def paginate_to_embeds(title: str,
                       .set_footer(text=f"page: {i + 1}/{len(paginator.pages)}"))
     return embeds
 
+def dict_to_fields(embed: Embed, 
+                   fields: Dict[str, str], 
+                   inline: bool=False) -> Embed:
+    """
+    Iterates through dict keys and appends them to the embed fields
+    Args:
+        embed: Embed object
+        dict: dict object that will be appended to the embed fields
```

Comment:
`dict` param in the docstring is different than in the function constructor. It would also be of use to specify that each key and value correspond to each field's name and value.